speechbrain/core.py

```diff
@@ -923,9 +929,10 @@ def check_gradients(self, loss):
                 return False
 
         # Clip gradient norm
-        torch.nn.utils.clip_grad_norm_(
-            (p for p in self.modules.parameters()), self.max_grad_norm
-        )
+        if self.max_grad_norm != 0.0:
```

Comment:
Since it wouldn't make sense to have a negative norm, perhaps a better comparison would be:
```suggestion
        if self.max_grad_norm > 0.0:
```

Reply:
Makes sense will do

Reply:
done